index.html

```diff
@@ -0,0 +1,357 @@
+            </div>
+            <div class="right">
+              <i class="icon">
+                <img src="./resources/like.svg" alt="options" />
+              </i>
+              <span>19K</span>
+              <i class="icon">
+                <img
+                  class="rotate180"
+                  src="./resources/like.svg"
+                  alt="options"
+                />
+              </i>
+              <span>DISLIKE</span>
+              <i class="icon">
+                <img src="./resources/share.svg" alt="options" />
+              </i>
+              <span>SHARE</span>
+              <i class="icon">
+                <img src="./resources/save.svg" alt="options" />
+              </i>
+              <span>SAVE</span>
+            </div>
+          </div>
+          <div class="chanelOwner">
+            <div class="chanel">
+              <div class="left">
+                <span class="chanelLogo icon">
+                  <img src="./resources/user.svg" alt="" />
+                </span>
+                <div>
+                  <p class="title">admas music</p>
+                  <p class="small">649k subscribers</p>
+                </div>
+              </div>
+              <div class="right">
+                <span><button class="subscribeButton">subscribe</button></span>
+                <span class="icon"
+                  ><img src="./resources/alarm.svg" alt=""
+                /></span>
+              </div>
+            </div>
+            <div class="more">
+              <p>
+                New Ethiopian Music 2021 : Traditional Amharic Eskista song Welo
+                Mejen (ወሎ መጀን) by Frezer
+              </p>
+              <p>Kenaw (ፍሬዘር ቀናው).</p>
+              <p>© 2021 Admas Entertainment. All Rights Reserved.</p>
+            </div>
+          </div>
+          <div class="comments">
+            <div class="review">
+              <span>1,795 Comments</span>
+              <span class="icon">
+                <img src="./resources/sorting.svg" alt="options" /> </span
+              ><span> SORT BY</span>
+            </div>
+            <div class="commentLists">
+              <div class="singleComment">
+                <div class="icon">
+                  <img src="./resources/userSpeaking.svg" alt="options" />
+                </div>
+                <div class="newComment"">
+                <input
+                  type="text"
+                  name="newComment"
+                  placeholder="Add a public comment..."
+                />
+                <div class="buttons">
+                  <button type="submit">CANCEL</button>
+                  <button type="submit">COMMENT</button>
+                </div>
+              </div>
+            </div>
+            <div class="singleComment">
+              <!-- <div class="icon">
+                <img src="./resources/coment-user1.svg" alt="options" />
+              </div>              <div class="icon"> -->
```

Comment:
- You should not leave commented codes in your code base, as this will clash with keeping your code clean.  You should remove commented code, and you can go back to your commit history to visit those.